backend/src/cusco/main.py

```
@@ -74,6 +113,22 @@ async def _bg_load_adc():
         logger.warning(f"Background AdC load failed (will retry on query): {e}")
 
 
+async def _bg_load_prr():
+    try:
+        await prr_source._ensure_loaded()
+        logger.info("PRR data loaded in background.")
+    except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)